ui-tests/src/test/java/io/syndesis/qe/fragments/common/menu/KebabMenu.java

```diff
@@ -32,4 +38,16 @@ public SelenideElement getItemElement(String item) {
         return menuButton.$(By.xpath("//a[text()='" + item + "']"));
     }
 
+    public void checkActionsSet(String[] expectedItems) {
+        Assertions.assertThat(getItemsStrings().containsAll(new HashSet<String>(Arrays.asList(expectedItems))));
+    }
+
+    private Set<String> getItemsStrings() {
+        Set<String> strings = new HashSet<String>();
```

Comment:
I'd prefer plain ArrayList or LinkedList as an output to keep posibility to test duplicates.

Reply:
I can do it. But the list should be Set in reality.
done

Reply:
Yes, and we should check the reality by assert probably :)

Reply:
It's a matter of a scenario. I don't want to do it right now in advance. But you're right.